CODE_OF_CONDUCT.md

```diff
@@ -2,7 +2,7 @@
 
 ## Our Pledge
 
-We as members, contributors, and leaders pledge to make participation in our
+We as members, contributors, and leaders pledge to participate in our
```

Comment:
We use "make participation" to imply "facilitating or ensuring someone's involvement.
This is different from directly "participate" as some roles.